src/constants/addresses.ts

```diff
@@ -129,6 +151,8 @@ export const NONFUNGIBLE_POSITION_MANAGER_ADDRESSES: AddressMap = {
   [SupportedChainId.CELO_ALFAJORES]: CELO_NONFUNGIBLE_POSITION_MANAGER_ADDRESSES,
   [SupportedChainId.OPTIMISM_GOERLI]: OPTIMISM_GOERLI_NONFUNGIBLE_POSITION_MANAGER_ADDRESSES,
   [SupportedChainId.ARBITRUM_GOERLI]: ARBITRUM_GOERLI_NONFUNGIBLE_POSITION_MANAGER_ADDRESSES,
+  [SupportedChainId.BOBA]: BOBA_NONFUNGIBLE_POSITION_MANAGER_ADDRESSES,
+  [SupportedChainId.BOBA_GOERLI]: BOBA_GOERLI_NONFUNGIBLE_POSITION_MANAGER_ADDRESSES,
 }
 
 export const ENS_REGISTRAR_ADDRESSES: AddressMap = {
```

Comment:
L167: lets add TICK_LENS_ADDRESSES for Boba and boba goerli

Reply:
It´s there.